src/requirementslib/models/requirements.py

```diff
@@ -887,9 +887,7 @@ def parsed_setup_cfg(self):
             and self.setup_cfg
         ):
             return {}
-        base_dir = os.path.dirname(os.path.abspath(self.setup_cfg))
-        setup_content = read_source(self.setup_cfg)
-        return parse_setup_cfg(setup_content, base_dir)
+        return self.setup_info.parse_setup_cfg()
```

Comment:
Will try `setuptools_parse_setup_cfg()` first, then the home grown version.

Reply:
+1 thanks for this